nx_parallel/algorithms/shortest_paths/weighted.py

```diff
@@ -1,57 +1,82 @@
-from joblib import Parallel, delayed
-from networkx.algorithms.shortest_paths.weighted import single_source_bellman_ford_path
+# TODO : update docstrings
+
+"""
+Shortest path parallel algorithms for weighted graphs.
+"""
 
+from joblib import Parallel, delayed
 import nx_parallel as nxp
+from networkx.algorithms.shortest_paths.weighted import (
+    single_source_dijkstra,
+    single_source_dijkstra_path_length,
+    single_source_dijkstra_path,
+    single_source_bellman_ford_path,
+    single_source_bellman_ford_path_length,
+)
+
+__all__ = [
+    "all_pairs_dijkstra",
+    "all_pairs_dijkstra_path_length",
+    "all_pairs_dijkstra_path",
+    "all_pairs_bellman_ford_path_length",
+    "all_pairs_bellman_ford_path",
+]
+
+
+def all_pairs_dijkstra(G, cutoff=None, weight="weight"):
+    """Find shortest weighted paths and lengths between all nodes."""
+
+    def _calculate_all_pairs_dijkstra_subset(n):
+        return (n, single_source_dijkstra(G, n, cutoff=cutoff, weight=weight))
```

Comment:
I think this should be `G.graph_object` passed into `single_source_dijkstra`. But maybe I'm not understanding...
```python
        return (n, single_source_dijkstra(G.graph_object, n, cutoff=cutoff, weight=weight))
```